content/en/docs/reference/using-api/health-checks.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Kubernetes API health endpoints
+reviewers:
+- logicalhan
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+This page describes the usage of the Kubernetes API endpoints for liveness and readiness.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## API Endpoints for health
```

Comment:
nit (feel free to merge as-is):
```suggestion
## API Endpoints for health
```

(because page subheadings are sentence case, and also to avoid confusion with [the resource named Endpoints](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#endpoints-v1-core)).

Reply:
I didn't see any difference here in the current state and you suggestions?

Reply:
Ah, I must have meant:
nit (feel free to merge as-is):
```suggestion
## API endpoints for health
```